LogMonitor/README.md

```diff
@@ -5,14 +5,12 @@
 **Log Monitor** is a log tool for Windows Containers. It monitors configured log sources and pipes a formatted output to a configured output location. Supported log sources include:

```

Comment:
We don't support anything other than STDOUT - perhaps change this to be specific till we have additional output locations